<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mantCategoria</class>
 <widget class="QMainWindow" name="mantCategoria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Categoria de Productos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtNombreCategoria">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>311</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblIdCategoria">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*Código           :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIdCategoria">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombreCategoria">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> *Categoría     :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombreCategoria_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>NUEVA CATEGORIA DE PRODUCTOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/error.png</normaloff>../icon/error.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/save.png</normaloff>../icon/save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtIdCategoria</tabstop>
  <tabstop>txtNombreCategoria</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
